test/extended/util/client.go

```diff
@@ -455,7 +455,7 @@ func (c *CLI) SetupWorkSpaceWithNamespace() {
 func (c *CLI) SetupWorkSpaceWithSpecificPath(serverURL string) {
 	newWorkSpace := names.SimpleNameGenerator.GenerateName(fmt.Sprintf("e2e-test-%s-", c.kubeFramework.BaseName))
 	e2e.Logf("Creating workspace %q", newWorkSpace)
-	output, errinfo := c.WithoutKubeconf().WithoutWorkSpaceServer().Run("ws").Args("create", "--server="+serverURL, newWorkSpace).Output()
+	output, errinfo := c.WithoutKubeconf().WithoutWorkSpaceServer().Run("kcp").Args("ws", "create", "--server="+serverURL, newWorkSpace).Output()
```

Comment:
@zimo-xiao do not think this change is required, in your system sym links might have been broken and that might be the reason you might be getting an error.

Reply:
something like below
ln -sf /home/kubectl-workspace /home/knarra/kubectl-ws
ln -sf /home/kubectl-workspace /home/knarra/kubectl-workspaces

Reply:
I agree with you and fixed my local link. I had reverted the change.